modules/social_features/social_comment/social_comment.info.yml

```diff
@@ -4,7 +4,8 @@ type: module
 core_version_requirement: ^9 || ^10
 dependencies:
   - drupal:comment
-  - group:group
   - social:activity_creator
   - social:social_core
+  - social:social_group
```

Comment:
@VBaldych Why do we need dependency from `social_group` here?
Maybe this is a reason why install script is failing.
Try to remove it and check if it fixes the issue.